### ISSUE TITLE
Confusion about order of input dimensions in SwinUNETR

### ISSUE THREAD

**Issue by HSILA** [Project-MONAI/research-contributions#244]
Consider the SwinUNETR model, the pertaining section. In [`utils/ops.py`](https://github.com/Project-MONAI/research-contributions/blob/main/SwinUNETR/Pretrain/utils/ops.py), one can have two interpretations of the order of dimensions of the input data after transformations. 

1- C,H,W,Z: as in [`patch_rand_drop`](https://github.com/Project-MONAI/research-contributions/blob/main/SwinUNETR/Pretrain/utils/ops.py#L18) method. 

2- C,Z,H,W: In the [`rot_rand`](https://github.com/Project-MONAI/research-contributions/blob/main/SwinUNETR/Pretrain/utils/ops.py#L46) method, the rotation (`x.rot90(1, (2, 3))`) is on axis (2,3), and it is mentioned in the paper that rotations are along the z-axis. 

I think since there is an `AddChanneld` transformation  in [`data_utils.py`](https://github.com/Project-MONAI/research-contributions/blob/main/SwinUNETR/Pretrain/utils/data_utils.py#L75), the second one should be correct, but `patch_rand_drop` consistently uses the first notation, I'm a little bit confused.

**Comment by sarvanichinthapalli** [comment]
Yes Same query because of that the image are changing from axial view to coronal view also which is not at all required
The repo is very badly maintained. Many of the important and valid queries are unanswered
